platform-db-runtime/src/main/java/com/softicar/platform/db/runtime/query/AbstractDbQuery.java

```diff
@@ -29,8 +29,7 @@
 	@Override
 	public DataTableIdentifier getIdentifier() {
 
-		String sql = new DbSqlFormatter(createSelect()).setExpandParameters(false).format();
-		return new DataTableIdentifier(sql);
+		return new DataTableIdentifier(getClass().getCanonicalName());
```

Comment:
This will lead to wrong results if an SQML query uses a static IF too add/remove columns, depending on a parameter. I think we need to create a story to fix this once and for all, and reference it here.